index.ts

```diff
@@ -524,6 +536,28 @@ class IGCParser {
 
     return extensions;
   }
+  private parseLRecord(line: string): IGCParser.LRecord {
+    let match = line.match(RE_L);
+
+    
+    // TODO: According to https://xp-soaring.github.io/igc_file_format/igc_format_2008.html#link_4.5 there are no L records allowed after a G record.
+    // But several fixtures violate this rule.
+    // What's the desired approach here?
+    // if (this._result.security) {
+    //   throw new Error(`No L record allowed past G record`);
+    // }
```

Comment:
according to the 2023 IGC spec:

> The logbook/comments (L) Record data may be placed anywhere after the H, I and J Records and there may be several L-record lines in the file.

![Bildschirmfoto 2025-01-06 um 15 43 45](https://github.com/user-attachments/assets/4b34d32f-43d5-4e9b-970c-68bbed5a5643)

and the example even has `L` records after `G` 😅